public/docs/_examples/structural-directives/dart/lib/structural_directives_component.dart

```diff
@@ -0,0 +1,21 @@
+// #docplaster
+// #docregion
+import 'package:angular2/angular2.dart';
+import 'unless_directive.dart';
+import 'heavy_loader_component.dart';
+
+@Component(
+    selector: 'structural-directives',
+    templateUrl: 'structural-directives.component.html',
+    styles: const ['button { min-width: 100px; }'],
+    directives: const [UnlessDirective, HeavyLoaderComponent])
+class StructuralDirectivesComponent {
+  List<String> heroes = ['Mr. Nice', 'Narco', 'Bombasto'];
+  bool condition = true;
+  bool isVisible = true;
+  List<String> logs = [];
+  String status = 'ready';
+
+  get hero => heroes.elementAt(0);
```

Comment:
I'm curious why you used `heroes.elementAt(0)` instead of `heroes[0]`. `heroes?.elementAt(0)` would make more sense to me...